### PR TITLE
Add GET /api/users endpoint

### PR DESCRIPTION
**Pull request by mxlro-star**
Add GET /api/users endpoint

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -74,5 +74,77 @@ describe("app", () => {
...
+          .then(({ body: { users } }) => {
+            expect(users).toHaveLength(4);
+            users.forEach((user) => {
+              expect(user).toHaveProperty("username");
+              expect(user).toHaveProperty("name");
+              expect(user).toHaveProperty("avatar_url");
+            });
+          });
+      });
+    });
```
Comment (Rayhaan2k9) [inline __tests__/app.test.js:148]: I would add a few more tests for if the inc_votes is not there so empty object passed in. Also for if the review doesn't exist like 9999 and if the review id is invalid like /api/reviews/banana

**Review by Rayhaan2k9 — CHANGES_REQUESTED** [review]
Almost there. Check comment :)
